lib/listings/ListingService.ts

```diff
@@ -76,6 +76,12 @@ class ListingService extends FirebaseService {
     const groupQuery = query(this.getCollectionGroupRef());
     return getDocs(groupQuery);
   }
+  async fetchcurrentuserListings(
```

Comment:
same here, id just called this `fetchUserListings`, the scope of this function is not only the current user